### ISSUE TITLE
Error when running the oomph

### ISSUE THREAD

**Issue by d-ruiz** [hengsin/idempiere-oomph#3]
Hi Heng Sin, 

I am getting this error when I run the installer:
Performing Git Clone git@github.com:idempiere/idempiere.git  (master)
Cloning Git repo git@github.com:idempiere/idempiere.git to /home/diego/hgIdempiere/masteridempiere/idempiere-ws/idempiere
java.lang.Exception: org.eclipse.jgit.api.errors.TransportException: git@github.com:idempiere/idempiere.git: Cannot log in at github.com:22
  at org.eclipse.oomph.setup.git.impl.GitCloneTaskImpl.perform(GitCloneTaskImpl.java:989)
  at org.eclipse.oomph.setup.internal.core.SetupTaskPerformer.doPerformNeededSetupTasks(SetupTaskPerformer.java:3851)
  at org.eclipse.oomph.setup.internal.core.SetupTaskPerformer.access$1(SetupTaskPerformer.java:3794)
  at org.eclipse.oomph.setup.internal.core.SetupTaskPerformer$WorkspaceUtil$1.run(SetupTaskPerformer.java:5178)
  at org.eclipse.core.internal.resources.Workspace.run(Workspace.java:2292)
  at org.eclipse.core.internal.resources.Workspace.run(Workspace.java:2317)
  at org.eclipse.oomph.setup.internal.core.SetupTaskPerformer$WorkspaceUtil.performNeededSetupTasks(SetupTaskPerformer.java:5172)
  at org.eclipse.oomph.setup.internal.core.SetupTaskPerformer$WorkspaceUtil.access$0(SetupTaskPerformer.java:5170)
  at org.eclipse.oomph.setup.internal.core.SetupTaskPerformer.performNeededSetupTasks(SetupTaskPerformer.java:3785)
  at org.eclipse.oomph.setup.internal.core.SetupTaskPerformer.performTriggeredSetupTasks(SetupTaskPerformer.java:3760)
  at org.eclipse.oomph.setup.internal.core.SetupTaskPerformer.perform(SetupTaskPerformer.java:3638)
  at org.eclipse.oomph.setup.ui.wizards.ProgressPage$9.run(ProgressPage.java:595)
  at org.eclipse.oomph.setup.ui.wizards.ProgressPage$11$1.run(ProgressPage.java:722)
  at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
 
Did I miss a step or what can be the source of this issue?

**Comment by hengsin** [comment]
Do you have github ssh access setup at your machine ?

I will clean up the repo and did a new release soon. In the new release, will change the default checkout protocol to https instead.

**Comment by hengsin** [comment]
Google is your friend :). Found the issue - https://www.eclipse.org/forums/index.php/t/1107389/

**Comment by d-ruiz** [comment]
I think I expressed myself poorly, my bad. 

The reason why I created the issue was that this is a much easier way to install iDempiere and that's something we're looking forward to, this should be the default in the near future. But I don't get why the installers attempt to log in to clone the repository, but with the change, you mention above, I think it will run smoothly. 

Thanks for working on this, a really nice tool.
